components/PlantCard.js

```diff
@@ -50,17 +68,21 @@ export default function PlantCard({ plant }) {
       <ImageWrapper>
         <Image
           alt={`Image of ${plant.name}`}
-          layout="fill"
+          fill
           src={
             plant.imageUrl ||
             "https://images.unsplash.com/photo-1564502983799-becfbf817b4f?q=80&w=2070&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
           }
         />
       </ImageWrapper>
-      <CardContent>
-        <CardPlantName>{plant.name}</CardPlantName>
-        <CardBotanicalPlantName>{plant.botanicalName}</CardBotanicalPlantName>
-      </CardContent>
+
+      <CardText>
+        <div>
```

Comment:
Which function fulfills this div? It seems like you use it as a wrapper around the plant names, is it important for your styling? On the other hand, the div isn't styled yet - this is a bit confusing.